app/views/partners/family_requests/confirmation.html.erb

```diff
@@ -1,26 +1,21 @@
-<section style="display:flex;flex-direction:column;align-items:center;padding-top:25px;padding-bottom:25px;">
-  <div style="width:700px;display:flex:flex-direction:row;align-items:center;justify-content:center;padding:10px;background-color:#fff;border: 1px solid rgb(213, 213, 213);box-shadow: 0 0.5px 3px 1px rgb(228, 228, 228);border-radius:10px">
-    <h3 style="font-size:2.5rem">Request Summary</h3>
-    <div style="display:flex;flex-direction:column; justify-content:space-evenly">
+<section class="d-flex flex-column align-items-center py-4">
+  <div class="card shadow mx-auto" style="width: 40rem;">
+    <div class="card-body">
+      <h1 class="text-center">Request Summary</h1>
+      <hr>
       <% @items.each do |item_name, child_info| %>
-        <p style="padding:0;margin:0;font-size:1.2rem"><%= item_name %> for</p>
-        <p style="padding:0;margin:0;font-size:1.2rem"><%= child_info[:first_name] %> <%= child_info[:last_name] %></p>
+        <p class="card-text">
+          <strong><%= item_name %></strong> for <%= child_info[:first_name] %> <%= child_info[:last_name] %>
+        </p>
         <hr>
       <% end %>
-    <div>
-
-     <h3 style="font-size:1.6rem">Please confirm that the above list is what you meant to request</h3>
-    <%= form_with(url: partners_family_requests_path, method: :post) do %>
-      <% @children_ids.each do |child_id| %>
-        <%= hidden_field_tag "child-#{child_id}", 'true' %>
-      <% end %>
-      <div style="display:flex;justify-content:space-between;align-items:center">
-        <%= link_to 'No, I need to change something.', 'javascript:history.back()', rel: 'nofollow', style: "font-size: 1.6rem;" %>
-      <%= button_tag(type: 'submit', class: 'btn btn-primary', style: "font-size: 1.6rem;") do %>
-        Yes, it's right
-        <% end %>
+      <h3 class="card-subtitle mb-4 text-center">Please confirm that the above list is what you meant to request</h3>
+      <div class="d-flex justify-content-between align-items-center">
+        <%= link_to 'No, I need to change something.', 'javascript:history.back()', rel: 'nofollow', class: "btn btn-danger" %>
+        <%= link_to 'Yes, it\'s right', partners_family_requests_path, method: :post, class: 'btn btn-primary' %>
```

Comment:
Is there a reason these are links and not buttons? At the very least the submit should be a button. I think it's OK to make it more concise, e.g. "Go Back" and "Submit".

Reply:
Ok, I was following original tickets language

Reply:
Kept the go back as a link to keep the easy javascript history.back, changed the submit's all to buttons